### DISCUSSION TITLE
Support for viewing the language that type from RTL

### DISCUSSION THREAD

**Discussion by Bunny-77X** [jocmp/capyreader#323] [Feature Requests]
Please support viewing languages that type from right to left. 

![Screenshot_20240904_224450.jpg](https://github.com/user-attachments/assets/d60a17a1-27a4-479d-8bb1-f6bcfc576f8d)

![Screenshot_20240904_224423.jpg](https://github.com/user-attachments/assets/c398246c-d354-449c-8e7e-c37c19f09241)

in Capy Reader. The Arabic text is displayed from left to right, which can make it challenging to read certain articles, especially when there are English words interspersed within the text. This can be quite confusing and hinder the reading experience.

Thank you ♡

**Comment by jocmp** [comment]
hey @Bunny-77X. RTL support is my next focus. I'm tracking it in this bug ticket https://github.com/jocmp/capyreader/issues/299

**Reply by Bunny-77X** [reply]
OK. Thank you so much ♡

Do you want me to delete this thread?

**Reply by jocmp** [reply]
No need to delete this post. I'll add an update here once it's finished. I've already fixed the reader page and I'm working on the article feed now. Both will be in the next release.

**Reply by jocmp** [reply]
@Bunny-77X This is live in [2024.09.1042-dev](https://github.com/jocmp/capyreader/releases/tag/2024.09.1042-dev). Let me know if you notice any formatting issues.

**Reply by Bunny-77X** [reply]
Okay 👍🏻

**Reply by Bunny-77X** [reply]
Hi @jocmp


I tried the update. The text direction is fixed. 

Thank you very much ♡

![Screenshot.jpg](https://github.com/user-attachments/assets/894a50c9-ba21-422e-b9b1-90e47f89fed6)


**But I found an error in displaying the date and the name of the author of the article. This error is related to the app language and the name of the author.**


Personally. I don't care if this bug is fixed or not, but I wanted to let you know in case you wanted to fix it.

**I marked the error with a red line.**


![20240910_021219_411.jpeg](https://github.com/user-attachments/assets/f3f65068-908f-4104-a155-1ae676cbe7e2)


![20240910_021023_201.jpeg](https://github.com/user-attachments/assets/de7e74c6-1a25-4639-aa9a-4661312d788e)

If the author's name is in English and the app is in Arabic, the author's name is displayed before the day and month.

------------------------

The same applies to the English language and the author's name in Arabic


![20240910_022142_043.jpeg](https://github.com/user-attachments/assets/96bac122-4a2a-48ca-bc19-137c8be9e6b9)



![20240910_022307_667.jpeg](https://github.com/user-attachments/assets/296d7ff0-38d4-49b1-8041-33ddb7aacd0f)

**Reply by jocmp** [reply]
I definitely want to make RTL work in all instances so this is great info. I believe this is how the markup is handled automatically. I'll see if I can make this follow RTL based on the app language too: https://github.com/jocmp/capyreader/issues/349
